### PR TITLE
Scaffold single work

### PR DESCRIPTION
**Pull request by mikeyrayvon**
make improvements and clean up a bit

### CODE REVIEW

File: lib/functions-custom.php [hunk 1]
```diff
@@ -148,18 +148,18 @@ function front_page_posts() {
 
 
 // Render related posts section from artist array
-function render_related_by_artist($artist_slug_array) {
+function render_related_by_artist($current_post_id) {
```
Comment (varffs) [inline lib/functions-custom.php]: function name is a bit confusing as this appears to be able to deal with more than one artist but the function is singular

File: single-work.php [hunk 2]
```diff
@@ -0,0 +1,74 @@
+<?php
+get_header();
+?>
+
+<main id="main-content">
+  <section id="posts">
```
Comment (varffs) [inline single-work.php:6]: single work shouldnt be in a `#posts` section

File: single-work.php [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+<?php
+if (have_posts()) {
+  while (have_posts()) {
+    the_post();
+
+    $work_cats = wp_get_post_terms($post->ID, 'work_cat');
+    $work_artists = get_post_artists($post->ID);
```
Comment (varffs) [inline single-work.php]: is this one of our utility functions? Perhaps we should start to prefix them `igv_` as it looks a bit like a core wp function so could send someone the wrong way to debug

File: single-work.php [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+?>
+            <div class="grid-item item-s-12 font-sans margin-bottom-basic"><?php echo $work_cats[0]->name; ?></div>
+<?php
+      }
+?>
+            <div class="grid-item item-s-10 item-l-6">
+              <h1 class="font-serif font-italic"><?php echo get_the_title($post->ID); ?></h1>
+
+<?php
+      if (!empty($work_artists)) {
```
Comment (varffs) [inline single-work.php:32]: Possibly the `bws-mx-design-work.png` file in final design folder shouldn't be there? This layout is the same as the pdf but doesn't match the png

File: single-work.php [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+?>
+            </div>
+            <div class="grid-item item-s-2 item-l-6 text-align-right">
+              Inquire
+            </div>
+          </div>
+
+          <div class="grid-row margin-bottom-basic">
+            <div class="grid-item item-s-12 item-l-6 font-sans">
+              <?php echo !empty($work_details) ? apply_filters('the_content', $work_details) : '&nbsp;'; ?>
```
Comment (varffs) [inline single-work.php]: What is the reason a nbsp is rendered if this is empty? The grid should still work without any content in the div I think?

File: single-work.php (hunk 2 above)
Comment (mikeyrayvon) [reply single-work.php:6]: what should it be in

Comment (mikeyrayvon) [reply single-work.php:6]: you know i've never actually used that ID for anything

File: single-work.php (hunk 4 above)
Comment (mikeyrayvon) [reply single-work.php:32]: the last 5 pages of that pdf is now blank for me. did you re-save it or something?

Comment (mikeyrayvon) [reply single-work.php:32]: but as i recall the work template wasn't included in the pdf

Comment (mikeyrayvon) [reply single-work.php:32]: yea the pdf is corrupted now

Comment (mikeyrayvon) [reply single-work.php:32]: i replaced it

Comment (mikeyrayvon) [reply single-work.php:32]: i think you were thinking the single-event template in the pdf was single-work. theyre similar

File: single-work.php (hunk 5 above)
Comment (mikeyrayvon) [reply single-work.php]: grid-items have flex-grow off by default. so if theyre empty they shrink down to nothing. the alternative here is adding `flex-grow` class to the grid item

File: single-work.php (hunk 2 above)
Comment (varffs) [reply single-work.php:6]: i personally think it should be either the wordpress name for the template aka `#single-work` or acceptably if the convention is here `#posttype-puralornah` aka `#work-single` or probably better just totally delete the whole element if it isn't needed

File: single-work.php (hunk 5 above)
Comment (varffs) [reply single-work.php]: I think probably the grid shouldn't work like that but id favor an extra class over the nbsp

File: single-work.php (hunk 2 above)
Comment (mikeyrayvon) [reply single-work.php:6]: seems like the wrong use for the `<section>` elem

Comment (mikeyrayvon) [reply single-work.php:6]: to me a `<section>` is more like the whole gallery. which has multiple rows, etc

Comment (mikeyrayvon) [reply single-work.php:6]: i'll make an issue for this to do theme-wide

**Pull request merged**